renderer/components/nettests/index.js

```diff
@@ -50,8 +52,8 @@ export const tests = {
   facebook_messenger,
   telegram,
   whatsapp,
-  ndt: web_connectivity,
-  dash: web_connectivity,
+  ndt,
+  dash,
```

Comment:
I believe you have a bunch of TODOs comments to zap, right?

Reply:
Zapped :zap: